recipes/finalize.rb

```diff
@@ -15,14 +15,29 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Restart supervisord
-service "supervisord" do
-  supports restart: true
-  action %i[enable start]
+case node['cfncluster']['cfn_scheduler']
+when 'slurm'
+  # disable supervisord when scheduler is slurm
+  service "supervisord" do
```

Comment:
user could use supervisord for other needs. I would rather skip the creation of the parallelcluster supervisord config rather than explicitly disabling and stopping the service

Reply:
also the master will most likely have a daemon for slurm as well, just a different one.